www/page_data.inc

```diff
@@ -69,9 +69,16 @@ function loadPageRunData($testPath, $run, $cached, &$requests = null, $options =
                 "loadEventEnd"
             );
             for ($i = 0; $i < count($ret); $i++) {
+
+              // are relative to the performance timing (navitation start)
+              // whereas request start time is based on the first request on
+              // the page. We need to reconsile start time by computing the
+              // offset and fixing page timing.
+              $offset = (double)$ret[$i]['date'] - (double)$ret2[$i]['date'];

```

Comment:
In the request parsing code, date and startTime are not set in the same way. You are computing the offset from the date, but I believe it should be created from startTime.